pipelines/nextflow/workflows/dumper_pipeline/main.nf

```diff
@@ -16,15 +16,21 @@
 
 // Predefine the files that can be dumped and the number of files for each of them
 default_selection_map = [
+    'agp': 1,
+    'annotation': 1,
     'seq_regions': 1,
     'events': 1,
     'genome_metadata': 1,
+    'gff3': 1,
     'stats': 3,
     'fasta_pep': 1,
-    'fasta_dna': 1
+    'fasta_dna': 1,
+    'seq_attrib': 1,
 ]
 default_selection = default_selection_map.keySet() as ArrayList
 params.db_list = ''
+params.server_url = ''
+params.host = ''
```

Comment:
Shouldn't all the params definitions with their default value be defined in the JSON schema and/or the `nextflow.config` file?

Reply:
Those are only here to ensure we don't get `none` values and raise a warning from the validation, they are not true "default" values.